spec/ameba/rule/layout/trailing_blank_lines_spec.cr

```diff
@@ -4,35 +4,55 @@ module Ameba::Rule::Layout
   subject = TrailingBlankLines.new
 
   describe TrailingBlankLines do
-    it "passes if there is no blank lines at the end" do
-      source = Source.new "no-blankline"
+    it "passes if there is a blank line at the end of a source" do
+      source = Source.new "a = 1\n", normalize: false
       subject.catch(source).should be_valid
     end
 
-    it "fails if there is a blank line at the end of a source" do
-      source = Source.new "a = 1\n \n ", normalize: false
-      subject.catch(source).should_not be_valid
-    end
-
```

Comment:
We also need a spec for the `a = 1\n  ` case (whitespace characters after the newline at the end of a source.

Reply:
It's not TrailingBlankLines case. You speak about TrailingWhitespace. In these tests we should only TrailingBlankLines class.

Reply:
My bad, you're totally right :)